src/main/java/org/terasology/signalling/components/SignalTimeDelayComponent.java

```diff
@@ -16,7 +16,10 @@
 package org.terasology.signalling.components;
 
 import org.terasology.entitySystem.Component;
-
+/**
+ * Responsible for keeping track of the time delay in a signal
+ * delaySetting is the amount of time it is being delayed in milliseconds
```

Comment:
Instead of describing the `delaySetting` at the class-level JavaDoc you can actually give it its own snippet. So it would end up looking like so:

```
/** the amount of time the signal is being delayed in milliseconds *
public long delaySetting;
```